operator/pkg/utils/utilities.go

```diff
@@ -6,13 +6,64 @@ package utils
 import (
 	"os"
 	"strings"
+	"reflect"
+	"math"
 )
 
 func IsPSPEnabled() bool {
 	value, exists := os.LookupEnv("ENABLE_VMWARE_PSP")
 	return exists && "true" == strings.TrimSpace(value)
 }
 
+func RangeInt(min, max, step int) []int {
+	size := int(math.Ceil(float64((max - min)) / float64(step)))
+	l := make([]int, size)
+	for i := 0; i < size; i++ {
+		l[i] = min + i * step
+	}
+	return l
+}
+
+func isArrayOrSlice(a interface{}) bool {
+	t := reflect.TypeOf(a)
+	k := t.Kind()
+	return k == reflect.Slice || k == reflect.Array
+}
+
+func DeepEqualArrayIgnoreOrder(a interface{}, b interface{}) bool {
```

Comment:
how do you feel about keeping the naming matching testify?

https://github.com/emou/testify/blob/8d9174bb2b778c662039a4dc921b505d14307a6b/assert/assertions.go#L738

Reply:
Sounds good.